ruby_review_week_four.rb

```diff
@@ -1,37 +1,95 @@
-# You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
+# # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
+
+# # 1. What is a variable?
+# A variable is a user-created term assigned to the value, where the value assigned may change thruoghout the program.
+
+# # 2. What Ruby method can you use to make a loop from an array?
+# <array name>.each do |value|
+# # 3. In Ruby, what are booleans?
+# True and False values
+
+# # 4. How do classes relate to objects?
+# Each object is an instance of a class
+# # 5. What is an instance variable?
+# A variable whose assigned value applies to a single object of a class. 
```

Comment:
yes, remember as well to use the word `attribute` here, as in an instance variable is an attribute of an object